cli/src/pcluster/api/util.py

```diff
@@ -154,6 +157,11 @@ def _deserialize_dict(data, boxed_type):
     return {k: _deserialize(v, boxed_type) for k, v in six.iteritems(data)}
 
 
+def assert_valid_node_js():
+    assert_node_executable()
+    assert_node_version()
+
+
 def assert_node_executable():
```

Comment:
minor: I'll add the underscore prefix to both `assert_node_executable` and `assert_node_version` to clarify they are "local" functions.